src/omg-cli/omg/commands/cleanup_project.go

```diff
@@ -90,28 +89,28 @@ func runAsync(steps []step, logger *log.Logger) error {
 	var errors []error
 	var errsMu sync.Mutex
 
-	for _, step := range steps {
+	for _, s := range steps {
 
 		wg.Add(1)
-		logger.Printf("running step %s asynchronously", step.name)
-		go func() {
-			if err := step.function(); err != nil {
-				logger.Printf("error running step %s: %v", step.name, err)
+		logger.Printf("running step %s asynchronously", s.name)
+		go func(s step) {
+			if err := s.function(); err != nil {
+				logger.Printf("error running step %s: %v", s.name, err)
 
 				errsMu.Lock()
 				errors = append(errors, err)
 				errsMu.Unlock()
 			}
 			wg.Done()
-		}()
+		}(s)
```

Comment:
Nice work, this should fix and issue I was running into today (cleanup not deleting the ops-man director, which would then start recreating the just deleted instances).